docs/user/quickstart.rst

```diff
@@ -354,10 +354,14 @@ But, since our ``status_code`` for ``r`` was ``200``, when we call
 ``raise_for_status()`` we get::
 
     >>> r.raise_for_status()
-    None
+    <Response [200]>
 
 All is well.
 
+.. note:: ``raise_for_status`` returns the response object for a successful response. This eases chaining in trivial cases:, where we want bad codes to raise an exception, but use the response otherwise:
```

Comment:
Nit: colon followed by comma is unnecessary. Remove the colon, as having two colons in one sentence is unwieldy.